src/AsyncSessionMiddlewareFactory.php

```diff
@@ -2,32 +2,39 @@
 
 namespace Interop\Session\Middleware\Async;
 
-use Psr\Http\Message\{ServerRequestInterface as Request, ResponseInterface as Response};
-use Interop\Session\Manager\{SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager};
+use Psr\Http\Message\{
+    ServerRequestInterface as Request, ResponseInterface as Response
+};
+use Interop\Session\Manager\{
+    SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager
+};
 use Interop\Session\Configuration\SessionConfigurationInterface;
 
 use Interop\Session\SessionInterface;
 
-class AsyncSessionMiddlewareFactory {
+class AsyncSessionMiddlewareFactory
```

Comment:
It is used to be userfriendly to container which does not support service-providers